tendrl/node_agent/persistence/disk.py

```diff
@@ -0,0 +1,39 @@
+from tendrl.common.etcdobj.etcdobj import EtcdObj
+from tendrl.common.etcdobj import fields
+
+
+class Disk(EtcdObj):
+    """A table of the Disk, lazily updated
+
+    """
+    __name__ = 'nodes/%s/Disks/%s/%s'
+
+    node_id = fields.StrField("node_id")
+    device_name = fields.StrField("device_name")
+    fs_type = fields.StrField("fs_type")
+    fs_uuid = fields.StrField("fs_uuid")
+    model = fields.StrField("model")
+    mount_point = fields.StrField("mount_point")
+    name = fields.StrField("name")
+    parent = fields.StrField("parent")
+    size = fields.StrField("size")
+    disk_type = fields.StrField("type")
+    used = fields.StrField("used")
+    ssd = fields.StrField("ssd")
+    vendor = fields.StrField("vendor")
+    disk_id = fields.StrField("disk_id")
+
+    def render(self):
+        if self.used == "True":
+            self.__name__ = self.__name__ % (
+                self.node_id,
+                "used",
+                self.disk_id
+            )
+        else:
+            self.__name__ = self.__name__ % (
+                self.node_id,
+                "free",
```

Comment:
I get the issue @r0h4n is talking about. But I understand API or UI layer needs list of un-unsed disks in most the cases like create cluster flow, add brick/osd flow etc.

So I feel we should have two different list APIs for used and free disks. But not sure how this would be achieved under one object definition only. I dont think we can have two `list` values defined for an object.

@r0h4n @brainfunked thoughts??